test/binary.go

```diff
@@ -0,0 +1,56 @@
+package integration
+
+import (
+	"context"
+	"fmt"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"log"
+)
+
+func validateBinaryConfig(integConfig IntegConfig) (string, string) {
+	s3Bucket, ok := integConfig["s3Bucket"].(string)
+	if !ok {
+		log.Fatal("Error: s3Bucket was not provided in integConfig.json")
+	}
+
+	cwaGithubSha, ok := integConfig["cwaGithubSha"].(string)
```

Comment:
This would generate different errors though. It would checking if the key exists and afterwards, checking if the value is string. So its not `cwaGithubSha was not provided in integConfig.json` but parsing `cwaGithubSha` failed.